ui/packages/image/src/ModifySketch.svelte

```diff
@@ -7,9 +7,10 @@
 </script>
 
 <div class="z-50 top-2 right-2 justify-end flex gap-1 absolute">
-	<IconButton Icon={Undo} on:click={() => dispatch("undo")} />
+	<IconButton Icon={Undo} aria-label="Undo" on:click={() => dispatch("undo")} />
```

Comment:
Follwing on from the above, when that is done we will need to change this to `label=...`.